src/core-manager/index.js

```diff
@@ -27,6 +27,10 @@ export const kCoreManagerReplicate = Symbol('replicate core manager')
  * @property {(coreRecord: CoreRecord) => void} add-core
  * @property {(namespace: Namespace, msg: { coreDiscoveryId: string, peerId: string, start: number, bitfield: Uint32Array }) => void} peer-have
  */
+/**
+ * TODO: Use a more precise type.
+ * @typedef {any} Peer
+ */
```

Comment:
I was adding a couple of new types that referenced peers. These are currently typed with `any`. Though I think we should use a more exact type in a followup PR, I think this makes it a little more clear.